src/pages/blogpostDetail/blogpostPage.jsx

```diff
@@ -0,0 +1,33 @@
+import {Link, useParams} from "react-router-dom";
+import posts from "/src/constants/data.json";
+import moment from "moment";
+
+function formatDate(dateString) {
+    const date = moment(dateString);
+    const formattedDate = date.format('DD MMMM YYYY');
+
+    return formattedDate;
+}
+
+function Blogpost() {
+    const {blogId} = useParams();
+
+
+    return (
+        <>
+        <h1>Blogpost Detail {blogId}</h1>
```

Comment:
je probeert in het h1 element een object te tonen, terwijl je hier eigenlijk een titel wilt tonen. Je moet hier dus {blogPost.title} gebruiken in plaats van enkel {blogPost}.